release.nix

```diff
@@ -35,31 +24,7 @@ let
         diff --unified --color=always README.md $out
       '';
 
-  validateExamples =
-    runCommand
-      "validate-examples"
-      ''
-        cd ${./.}
-        mkdir $out
-        LC_ALL=en_US.UTF-8 ./scripts/build-examples.py $out
-      '';
-
-  # Derivation that trivially depends on the input source code revision.
-  # As this is included in the "dhall-lang" aggregate, it forces every
-  # commit to have a corresponding GitHub status check, even if the
-  # commit doesn't make any changes (which can happen when merging
-  # master in).
-  rev = pkgs.runCommand "rev" {} ''echo "${src.rev}" > $out'';
```

Comment:
I think we need this for Hydra to always report builds?

Reply:
Ah good point.